src/tools/web/README.md

```diff
@@ -0,0 +1,37 @@
+# elektra web
+
+_an API and web user interface to remotely manage multiple elektra instances_
+
+
+## Overview
+
+Elektra web consists of multiple components:
+
+ * (multiple) servers running an elektra daemon ([`elektrad`](elektrad/))
+ * a single cluster management server to communicate with the elektra daemons ([`clusterd`](clusterd/))
```

Comment:
I'm not sure if recursive nesting is such a common usecase and I want to focus on getting one-level clusters working properly (main use case), so I'll leave this out for now. If I were to implement it, I'd do it with a single `clusterd` instance, though (allowing clusters to have clusters as their children).